scripts/lib/authority-and-program-updater.ts

```diff
@@ -139,17 +128,15 @@ export function updateAuthorities(
   }
 
   for (const [authorityShort, authority] of nonUtilities) {
-    if (
-      json[stateUpper][authority.authority_type.toLowerCase()] === undefined
-    ) {
-      json[stateUpper][authority.authority_type.toLowerCase()] = {};
+    if (json[authority.authority_type.toLowerCase()] === undefined) {
+      json[authority.authority_type.toLowerCase()] = {};
     }
-    json[stateUpper][authority.authority_type.toLowerCase()][authorityShort] = {
+    json[authority.authority_type.toLowerCase()][authorityShort] = {
       name: authority.name,
     };
   }
 
-  json[stateUpper].utility = existingUtilities;
+  json.utility = existingUtilities;
```

Comment:
nit (and unrelated to this change): let's move this line further up in this method, since it's a bit odd to have the non-utility authorities logic sandwiched in between. I think either before the partitioning or after the error-checking are both fine.